app/src/main/java/com/example/caloriecounter/feature/restaurant/ui/meals/MealItemComponent.kt

```diff
@@ -0,0 +1,70 @@
+package com.example.caloriecounter.feature.restaurant.ui.meals
+
+import androidx.compose.foundation.layout.*
+import androidx.compose.material.Card
+import androidx.compose.material.ExperimentalMaterialApi
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import coil.compose.AsyncImage
+import coil.request.ImageRequest
+import com.example.caloriecounter.R
+import com.example.caloriecounter.ui.theme.CalorieCounterTheme
+import com.example.domain.models.Meal
+
+@OptIn(ExperimentalMaterialApi::class)
+@Composable
+fun MealItemComponent(
+    meal: Meal,
```

Comment:
Should we pass a whole `Meal` model if we use only 3 of its properties? It can lead to redundant recompositions.